Cartfile

```diff
@@ -1,2 +1,2 @@
 github "raphaelmor/Polyline" ~> 4.2
-github "mapbox/turf-swift" ~> 0.3
+github "mapbox/turf-swift" ~> 0.3.0
```

Comment:
We need to specify the patch version for any pre-v1.0 dependency. Otherwise, both Carthage and CocoaPods will automatically pull down a newer minor version, in this case v0.5.0, causing lots of compiler errors.

We weren’t seeing the issue locally in this repository because of the Cartfile.resolved file. However, Carthage doesn’t look at that file when installing dependencies for downstream projects.